lib/index.js

```diff
@@ -296,9 +309,26 @@ exports.parseGeneral = BBPromise.method(function(chtml){
 	// Copy key-value pairs with defined values to meta
 	var meta = {};
 	var value;
+	var notEmpty = false;
```

Comment:
You have to set this as false inside the Object.keys().forEach() function. Otherwise this could be set as true randomly inside the function as each function inside forEach is run asynchronously.